src/components/DateRangePicker.jsx

```diff
@@ -247,13 +255,28 @@ class DateRangePicker extends React.PureComponent {
   }
 
   setDayPickerContainerRef(ref) {
+    if (this.dayPickerContainer) this.removeDayPickerEventListeners();
+    if (!ref) return;
     this.dayPickerContainer = ref;
+    this.addDayPickerEventListeners();
   }
 
   setContainerRef(ref) {
     this.container = ref;
   }
 
+  addDayPickerEventListeners() {
+    this.removeDayPickerFocusOut = addEventListener(
```

Comment:
It would be great if we could add a comment here as to why we're using this approach rather than a React event listener!

Reply:
Sure!